CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- Support comma+space-separated headers in check-http.rb
+- Support POST requests in check-http.rb
```

Comment:
I'd gotten into the habit of only listing changes that affect production in the changelog. It's updated now.